client/app/vps/cloud-database/vps-cloud-database.routing.js

```diff
@@ -1,15 +1,15 @@
+import { CLOUD_DATABASE_FEATURE } from './vps-cloud-database.constants';
+
 import component from './vps-cloud-database.component';
 
 export default /* @ngInject */($stateProvider) => {
   $stateProvider
     .state('iaas.vps.detail.cloud-database', {
       resolve: {
-        shouldNotBeCA: /* @ngInject */ (
+        isAuthorized: /* @ngInject */ (
           $q,
-          CucFeatureAvailabilityService,
-        ) => CucFeatureAvailabilityService
-          .hasFeaturePromise('VPS', 'cloudDatabase')
-          .then(hasFeature => (hasFeature ? $q.when() : $q.reject('Not authorized'))),
+          capabilities,
+        ) => (capabilities.includes(CLOUD_DATABASE_FEATURE) ? $q.when() : $q.reject('Not authorized')),
```

Comment:
Are we sure `capabilities` cannot be null ?

Reply:
Yes, it will be at least an empty `array`